e2e/__tests__/confirm_all_repos.spec.ts

```diff
@@ -26,3 +45,15 @@ test('Confirm all repos display', async ({ page }) => {
   // doesn't currently change page as blob parameters required
   //await page.locator('[id="IQEngineLogo"]').click();
 });
+
+test.afterAll(async ({}) => {
```

Comment:
Should this be afterEach?

Reply:
Or are we not altering the data in any way?

Reply:
The data is not being altered. It's there only so that the API-sourced datasource appears in the cards on the home page.